server/constants.js

```diff
@@ -0,0 +1,21 @@
+//All Event Handlers and Event Emitters Definitions
+const CHAT_EVENTS = {
+    NEW_EVENT_CLOSE: 'close',
+    NEW_EVENT_INACTIVE: 'inactive',
+    NEW_EVENT_DELETE_MESSAGE: 'delete_message',
+    NEW_EVENT_EDIT_MESSAGE: 'edit_message',
+    NEW_EVENT_JOIN: 'join',
+    NEW_EVENT_JOINED: 'joined',
+    NEW_EVENT_CHAT_RESTORE: 'chat_restore',
+    NEW_EVENT_LOGOUT: 'logout',
+    NEW_EVENT_SEND_MESSAGE: 'send_message',
+    NEW_EVENT_SEND_FAILED: 'send_failed',
+    NEW_EVENT_RECEIVE_MESSAGE: 'receive_message',
+    NEW_EVENT_STOP_SEARCH: 'stop_search',
+    NEW_EVENT_STOP_SEARCH_SUCCESS: 'stop_search_success',
+    NEW_EVENT_TYPING: 'typing',
+    NEW_EVENT_DISPLAY: 'display',
+    NEW_EVENT_ADDING: 'adding',
+    NEW_EVENT_CREATE_ROOM: 'createRoom'
+}
+module.exports = { CHAT_EVENTS }
```

Comment:
won't `module.exports = CHAT_EVENTS` give a much cleaner code. That way, you just use `const {NEW_EVENT_ADDING}`